### PR TITLE
improve ut for binding

### PR DESCRIPTION
**Pull request by AllenZMC**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
improve ut for binding
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by AllenZMC** [comment]
@XiShanYongYe-Chang

**Review by XiShanYongYe-Chang — COMMENTED** [review]
thanks~
/lgtm

**Comment by AllenZMC** [comment]
@RainbowMango

**Comment by RainbowMango** [comment]
/assign
Will look at it later. Thanks.

File: pkg/util/helper/binding_test.go [hunk 1]
```diff
@@ -180,3 +182,96 @@ func TestObtainBindingSpecExistingClusters(t *testing.T) {
...
+			name: "condition type is not Scheduled",
+			status: &workv1alpha2.ResourceBindingStatus{
+				Conditions: []metav1.Condition{
+					{Type: clusterapis.ClusterConditionReady, Status: metav1.ConditionTrue},
+				},
+			},
+			want: false,
+		},
+		{
+			name: "condition status is not true",
```
Comment (RainbowMango) [inline pkg/util/helper/binding_test.go:202]: `not true`?

File: pkg/util/helper/binding_test.go [hunk 2]
```diff
@@ -180,3 +182,96 @@ func TestObtainBindingSpecExistingClusters(t *testing.T) {
...
+			want: []ClusterWeightInfo{
+				{
+					ClusterName: "a",
+					Weight:      1,
+				},
+				{
+					ClusterName: "b",
+					Weight:      1,
+				},
+			},
```
Comment (RainbowMango) [inline pkg/util/helper/binding_test.go:267]: The `map` is unordered, which means we can't assume always get `a` then get `b`.  
I don't know how to test the `same weight` scenario.

Comment (RainbowMango) [inline pkg/util/helper/binding_test.go:267]: The `map` is unordered, which means we can't assume always get `a` then get `b`.  
I don't know how to test the `same weight` scenario.

**Pull request closed without merging**